README.md

```diff
@@ -6,6 +6,11 @@
 - 📫 How to reach me: emmanuel.eban@conicapps.com
 - 🤝 Connect with me on LinkedIn: <https://www.linkedin.com/in/eebann/>
 
-<!---
-available!!
---->
+## 📈 GitHub Stats
+
+  <div align="right">
```

Comment:
```suggestion
  <div align="center">
```

Reply:
nah it is intentional

Reply:
If you center it, it'll look like this or similar.

![image](https://user-images.githubusercontent.com/51391473/147366160-f734d3cf-ff7b-4a6d-bb24-268ec9cdfdcb.png)

What do you think?